plugins/generator-1.19.4/forge-1.19.4/templates/overlay.java.ftl

```diff
@@ -129,4 +122,26 @@ package ${package}.client.screens;
 	}
 
 }
+
+<#macro calculatePosition component>
+	<#if component.anchorPoint.name() == "TOP_LEFT">
+		${component.x}, ${component.y}
+	<#elseif component.anchorPoint.name() == "TOP_CENTER">
+		w / 2 + ${component.x - 213}, ${component.y}
+	<#elseif component.anchorPoint.name() == "TOP_RIGHT">
+		w - ${427 - component.x}, ${component.y}

```

Comment:
Just wondering, would it be possible to acquire numbers such as 427, 213, ... from some method or field from the Screen, similar we do with w and h?

Reply:
I don't think so. It seems to be the default width and height of minecraft in window-mode (not maximized) and also the size of the overlay-editor in mcreator.
Maybe replace them with constants either in template or code!?

Reply:
I see, in this case, leave them as it is